qa/rpc-tests/nov152018_forkactivation.py

```diff
@@ -0,0 +1,251 @@
+#!/usr/bin/env python3
+# Copyright (c) 2018 The Bitcoin developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""
+This test checks that simple features of the magnetic anomaly fork
+activates properly. More complex features are given their own tests.
+"""
+
+from test_framework.test_framework import ComparisonTestFramework
+from test_framework.util import assert_equal, assert_raises_rpc_error
+from test_framework.comptool import TestManager, TestInstance, RejectResult
+from test_framework.blocktools import create_coinbase, create_block
+from test_framework.mininode import *
+from test_framework.script import *
+from test_framework.cdefs import MIN_TX_SIZE
+from collections import deque
+
+# far into the future
+MAGNETIC_ANOMALY_START_TIME = 2000000000
+
+
+class PreviousSpendableOutput():
+
+    def __init__(self, tx=CTransaction(), n=-1):
+        self.tx = tx
+        self.n = n  # the output we're spending
+
+
+class MagneticAnomalyActivationTest(ComparisonTestFramework):
```

Comment:
I would also change the name of this `class` since we are not using ABC nomenclature.